src/sections/hero.html

```diff
@@ -0,0 +1,53 @@
+
+<section
+      class="overflow-hidden  ojemba-gradient1 font-railwayBold"
+   >
+      <nav class="flex justify-between text-xs text-white lg:text-xs">
+        <div class= "m-4 mt-8 lg:h-16 lg:ml-16 lg:w-52 w-14 h-7">
+          <img src="../../images/Logo.webp" alt="ojemba-logo" />
+        </div>
+        <div class="flex gap-2 lg:justify-between">
+          <img src="../../images/hamburger 1.webp" alt="" class="w-6 h-6 m-5 lg:hidden">
```

Comment:
Please ensure that the image naming convention does not include spaces between characters. I recommend using hyphens, underscores, or any other method that provides a consistent naming format.